### PR TITLE
fix: Include option in `getObject` feature is not working

### PR DESCRIPTION
**Pull request by mbfakourii**
### New Pull Request Checklist
- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Flutter/security/policy).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/Parse-SDK-Flutter/issues?q=is%3Aissue).

### Issue Description
fix include in get object

Closes: #813

### Approach
Use Parse SDK built-in functions to solve the problem

### TODOs before merging
- [ ] Add tests
- [ ] Add changes to documentation (guides, repository pages, in-code descriptions)
- [ ] A changelog entry

### CODE REVIEW

**Review by mtrezza — COMMENTED** [review]
Looks good, could you add the changelog and version bump?

**Comment by mbfakourii** [comment]
> Looks good, could you add the changelog and version bump?

added

File: packages/dart/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## [3.1.9](https://github.com/parse-community/Parse-SDK-Flutter/compare/dart-3.1.8...dart-3.1.9) (2022-12-24)
```
Comment (mtrezza) [inline packages/dart/CHANGELOG.md:1]:
```suggestion
## [3.1.9](https://github.com/parse-community/Parse-SDK-Flutter/compare/dart-3.1.8...dart-3.1.9) (2022-12-25)
```

**Review by mtrezza — DISMISSED** [review]
Looks good!

Edit: just noticed after merging that this should have been under "Bug Fixes" not "Features" in the changelog; I'll fix that separately.

**Pull request merged**